station.go

```diff
@@ -481,3 +498,77 @@ func (sd *schemaDetails) validJsonSchemaMsg(msg any) ([]byte, error) {
 
 	return msgBytes, nil
 }
+
+// [
```

Comment:
Please notice because it happens all the time. Remove comments.
PR sould be opened after you checked your code at least few times